blog/models.py

```diff
@@ -75,3 +79,25 @@ class UserVote(models.Model):
 
     class Meta:
         unique_together = ('user', 'blog')
+
+
+class Profile(models.Model):
+    user = models.OneToOneField(User, related_name='profile', on_delete=models.CASCADE)
+    GENDER_CHOICES = (
```

Comment:
Constants should be declared either at the start of file code (after imports) or at the top in the class.

Reply:
Done.